ci/pix4d_demo_pipeline.yml

```diff
@@ -0,0 +1,30 @@
+resource_types:
```

Comment:
Don't think this should be added either, or at least it's not obvious why this is being added as part of this PR.